src/frontend/src/components/items/StylishText.tsx

```diff
@@ -1,4 +1,4 @@
-import { Text } from '@mantine/core';
+import { Accordion, Text } from '@mantine/core';
```

Comment:
Q: Is this import really needed?

Reply:
No, that looks like it has crept in by mistake, I'll fix that